### PR TITLE
Copy update on charmed kubeflow homepage

### PR DESCRIPTION
**Pull request by bethcollins92**
## Done

- Copy update on index.html
- Update navigation

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8046
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check page against [copy doc](https://docs.google.com/document/d/1Wvhl0yYV_w0BJsyZbQFKg_QV6RjMLbmQU3x0rsD2F4Q/edit#heading=h.ohv58j8gh1iz)

## Issue / Card

Fixes [#8717](https://github.com/canonical-web-and-design/ubuntu.com/issues/8717)

## Screenshots

image

### CODE REVIEW

**Comment by pmahnke** [comment]
@bethcollins92 I asked @waynecrosby about the image... also Rui about the use of Hadoop...

File: templates/index.html [hunk 1]
```diff
@@ -112,7 +113,7 @@ <h2>Kubeflow, well packaged</h2>
     <div class="col-5 u-align--center">
       {{
         image(
-          url="https://assets.ubuntu.com/v1/7c6d52ef-Kubeflow-nicely-packaged-box.svg",
+          url="https://assets.ubuntu.com/v1/20f5bfae-Kubeflow-nicely-packaged-simple-transparent.svg",
           alt="",
           width="208",
```
Comment (pmahnke) [inline templates/index.html:118]:
```suggestion
          width="212",
```

File: templates/index.html [hunk 2]
```diff
@@ -112,7 +113,7 @@ <h2>Kubeflow, well packaged</h2>
...
           height="197",
```
Comment (pmahnke) [inline templates/index.html:119]:
```suggestion
          height="201",
```

**Comment by bethcollins92** [comment]
@pmahnke Updated with new image and buttons

**Review by pmahnke — APPROVED** [review]
just a few inline changes

**Pull request merged**